src/index.js

```diff
@@ -44,10 +44,9 @@ export ContinuousColorLegend from 'legends/continuous-color-legend';
 export ContinuousSizeLegend from 'legends/continuous-size-legend';
 
 export Table from 'table';
```

Comment:
these exports are not in alphabetical order, please put them in to alphabetical order